src/mapeo-project.js

```diff
@@ -558,20 +558,6 @@ export class MapeoProject extends TypedEmitter {
     return this.#roles.getRole(this.#deviceId)
   }
 
-  /**
-   * @param {string} createdBy The `createdBy` value from a document.
-   * @returns {Promise<string>} The device ID for this creator.
-   * @throws When device ID cannot be found.
-   */
-  async $createdByToDeviceId(createdBy) {
-    const discoveryKey = Buffer.from(createdBy, 'hex')
-    const coreId = this.#coreManager
-      .getCoreByDiscoveryKey(discoveryKey)
-      ?.key.toString('hex')
-    if (!coreId) throw new Error('NotFound')
-    return this.#coreOwnership.getOwner(coreId)
-  }
-
```

Comment:
What's the replacement for this? Don't we need a `$originalVersionIdToDeviceId` for now? Ideally we just append `createdBy` to the read value of observations, but in the meantime I think the frontend relies on this function?

Reply:
Replaced in ee39a1fccbdfd6ea608cfac15f418d0659117719.